components/button/button.css

```diff
@@ -1,30 +1,68 @@
-.storybook-button {
-  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-  font-weight: 700;
-  border: 0;
-  border-radius: 3em;
-  cursor: pointer;
-  display: inline-block;
-  line-height: 1;
-}
-.storybook-button--primary {
-  color: white;
-  background-color: #1ea7fd;
-}
-.storybook-button--secondary {
-  color: #333;
-  background-color: transparent;
-  box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 1px inset;
-}
-.storybook-button--small {
-  font-size: 12px;
-  padding: 10px 16px;
-}
-.storybook-button--medium {
-  font-size: 14px;
-  padding: 11px 20px;
-}
-.storybook-button--large {
-  font-size: 16px;
-  padding: 12px 24px;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .c-button {
+    @apply rounded px-8 py-4 min-w-[250px] bg-gray-100 flex items-center justify-center;
+    font-size: 16px;
+    line-height: 16px;
+  }
+
+  .c-button:disabled {
+    @apply bg-gray-200;
+    @apply text-gray-300;
+    cursor: not-allowed;
```

Comment:
We created an issue (https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/5) to implement the final colors, typography and spacing to the button component. The tailwind configuration will be implemented in this task (https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/3)